pootle/apps/pootle_store/unit/altsrc.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from pootle.i18n.gettext import language_dir
+from pootle_store.templatetags.store_tags import pluralize_target
+
+from .proxy import UnitProxy
+
+
+class AltSrcUnitProxy(UnitProxy):
+    @property
+    def store_path(self):
+        """Returns just the path part omitting language and project codes."""
+        return self.unit["store__pootle_path"].split(u'/', 2)[-1]
```

Comment:
Hm, does this do what you want?
```python
In [1]: '/ru/project/blah/bleh/meh.po'.split(u'/', 2)[-1]
Out[1]: u'project/blah/bleh/meh.po'
```

If not, I'd suggest to use `pootle.core.url_helpers.to_tp_relative_path` instead:
```python
In [2]: from pootle.core.url_helpers import to_tp_relative_path

In [3]: to_tp_relative_path('/ru/project/blah/bleh/meh.po')
Out[3]: u'blah/bleh/meh.po'
```